src/SprykerEco/Zed/PunchoutCatalogs/Business/Reader/PunchoutCatalogsReader.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\PunchoutCatalogs\Business\Reader;
+
+use Generated\Shared\Transfer\PunchoutCatalogConnectionTransfer;
+use SprykerEco\Zed\PunchoutCatalogs\Dependency\Facade\PunchoutCatalogsToVaultFacadeInterface;
+use SprykerEco\Zed\PunchoutCatalogs\Persistence\PunchoutCatalogsRepositoryInterface;
+
+class PunchoutCatalogsReader implements PunchoutCatalogsReaderInterface
+{
+    protected const VAULT_DATA_TYPE_PASSWORD = 'pwg_punchout_catalog_connection.password';
```

Comment:
`VAULT_DATA_TYPE_PASSWORD` feels disconnected from `pwg_punchout_catalog_connection.password`.
It is possible to fix this by renaming a constant?

Reply:
It shouldn't have the same name as `pwg_punchout_catalog_connection.password`.
Here `pwg_punchout_catalog_connection` it's just a prefix.

But agree, the name could be better.

Renamed to `PASSWORD_VAULT_DATA_TYPE`

Reply:
Answered here https://github.com/spryker-eco/punchout-catalogs/pull/4#discussion_r289934676